### PR TITLE
Add cosmosDb support and modify validate command

### PR DESCRIPTION
**Pull request by chagong**
Add cosmosDb support and modify validate command.

### CODE REVIEW

File: azext_connect/custom.py [hunk 1]
```diff
@@ -483,38 +486,18 @@ def bind_function(
...
-        target = _get_target_id(scope, mysql=mysql, database=database)
-        result = _validate(
-            cmd, subscription, resource_group, name, source, target, authtype='Secret', username=username, password=password
-        )
-        print(json.dumps(result, indent=2))
+        api = _create_api(cmd)
+        result = api.validate(subscription, resource_group, name)
+        if result.ok is not True:
+            end = result.text.find('\r\n\r\nHEADERS\r\n=======')
+            msg = result.text[:end] if end > -1 else result.text
```
Comment (dooriya) [inline azext_connect/custom.py:496]: Why find "HEADER" in the result?
I think we can check the validation result response body if the connection is available

Comment (adashen) [reply azext_connect/custom.py:496]: This is a workaround for the current error handling message, which out put all the request payload. So just remove the request payload.

Comment (dooriya) [reply azext_connect/custom.py:496]: OK~

File: azext_connect/commands.py [hunk 2]
```diff
@@ -5,10 +5,12 @@ def load_command_table(self, _):
 
     with self.command_group('cupertino webapp') as g:
         g.custom_command('bind', 'bind_webapp')
+        g.custom_command('validate', 'validate_general')
```
Comment (adashen) [inline azext_connect/commands.py]: the validate command which only need connection name is not bond to any service right? So put in command_group "cupertino" ?

File: azext_connect/custom.py (hunk 1 above)
Comment (chagong) [reply azext_connect/custom.py:496]: This part is for when result is not ok which means connection is not available.

File: azext_connect/commands.py (hunk 2 above)
Comment (chagong) [reply azext_connect/commands.py]: ah, making great sense. let me do that.

**Review by adashen — APPROVED** [review]

**Pull request merged**